src/include/distributed/multi_physical_planner.h

```diff
@@ -337,6 +337,13 @@ typedef struct Task
 	 * NULL, in which case executor might use a default destination.
 	 */
 	struct TupleDestination *tupleDest;
+
+	/*
+	 * EXPLAIN ANALYZE output fetched from worker. This is saved to be used later
+	 * by RemoteExplain().
+	 */
+	StringInfo fetchedExplainAnalyzePlan;
```

Comment:
I'm wondering whether there is a way to avoid putting execution information in the Task, which is meant to represent a part of a plan. I'm guessing there isn't, since we need to pass the information via the explain hooks.

Reply:
I suppose we could interpret remote explain plans as being part of the distributed plan (has valuable cost information).